test/install.mk

```diff
@@ -129,10 +129,15 @@ install-policy:
 	kubectl wait deployments istio-policy -n ${ISTIO_NS} --for=condition=available --timeout=${WAIT_TIMEOUT}
 
 # This target should only be used in situations in which the prom operator has not already been installed in a cluster.
+install-prometheus-operator: PROM_OP_NS="prometheus-operator"
 install-prometheus-operator:
-	# installs the prom operator in the default namespace
-	kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
-	kubectl wait --for=condition=available --timeout=${WAIT_TIMEOUT} deploy/prometheus-operator
+	kubectl create ns ${PROM_OP_NS} || true
```

Comment:
Do you want to add this to the install.sh script, and maybe make it the default ?

Reply:
If you don't mind me asking, what is the distinction between `install.sh` and the make files?

Reply:
I think this is a good point. AFAIK originally `install.sh` was intended as a simple way to install for demo usages. Now we have the `kubectl -k ...` command for that, so do we need the script?